plugin.onedrive/entrypoint.py

```diff
@@ -0,0 +1,21 @@
+#-------------------------------------------------------------------------------
```

Comment:
you might want to look into spdx headers in the future (look at the kodi source code), it makes this far less verbose :)